app/assets/stylesheets/application.css.erb

```diff
@@ -657,46 +657,51 @@ ol.comments1 > li.comments_subtree > ol.comments {
 }
 
 div.voters {
+  display: flex;
 	float: left;
 	margin-top: 0px;
 	width: 40px;
+	height: 40px;
 }
 
-div.voters div.score {
-	color: var(--color-fg-contrast-4-5);
-	font-size: 9pt;
-	margin-top: 1px;
-	margin-bottom: -3px;
-	text-align: center;
+div.voters span.score {
+  color: var(--color-fg-contrast-4-5);
```

Comment:
We use tab indents in this css file, so please review the changed lines in this file.